src/Cake.AzureDevOps/Cake.AzureDevOps.csproj

```diff
@@ -49,6 +49,10 @@
     <None Include="icon.png" Pack="true" PackagePath="\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Pipelines\TestManagementClient\" />
```

Comment:
Is this required?

Reply:
TestManagementClient is built the same way as BuildClient. This is the reason why it is done this way. It is ready for testing, but currently not used.

Reply:
It still should not be required to manually include the folder. The files in this folder should be automatically added to the solution.